packages/yarnpkg-types/package.json

```diff
@@ -29,5 +29,8 @@
   ],
   "engines": {
     "node": ">=14.15.0"
+  },
+  "dependencies": {
+    "tslib": "^2.4.0"
```

Comment:
Do we actually need tslib?

Reply:
TypeScript emits code that imports it so yes.

Reply:
Ah right, due to the `import type` not being marked as type anymore